test/controllers/ErrorTest.scala

```diff
@@ -32,7 +32,8 @@ class ErrorTest extends UnitSpec with TestUtility with GuiceOneAppPerSuite {
 
   "Checking user record" should {
 
-    "return BadRequest if there is an error while finding cid for recipient" in {
+    //TODO this needs investigating
```

Comment:
Unaddressed TODOs

Reply:
This will be investigated as part of the test framework rewrite